data/terraform_aws_resources.txt

```diff
@@ -0,0 +1,1295 @@
+aws_accessanalyzer_analyzer
```

Comment:
Really cool! What do you think about also adding Azure and GCP resources and deciding which cloud provider to go with at the start of a module run? It would be quite a bit more variety then.